docs/dev/reference/glossary/pi.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Pi
+id: pi
+full_link: https://www.raspberrypi.com/
```

Comment:
usually this is to a viam docs page.... this feels like we're advertising pis too much